Assets/MixedRealityToolkit/_Core/Utilities/Scripts/Int3.cs

```diff
@@ -1,249 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Thank you for getting rid of this file.... it always makes me think that it should break us into the debugger (yes, i know i am old.... )

Reply:
Haha, yea `Vector3Int` should be used instead.